packages/camera/camera_avfoundation/example/ios/RunnerTests/AvailableCamerasTest.m

```diff
@@ -73,6 +73,77 @@ - (void)testAvailableCamerasShouldReturnAllCamerasOnMultiCameraIPhone {
     XCTAssertTrue([dictionaryResult count] == 3);
   }
 }
+- (void)testAvailableCamerasShouldReturnExternalCameraIfAvailable {
+  CameraPlugin *camera = [[CameraPlugin alloc] initWithRegistry:nil messenger:nil];
+  XCTestExpectation *expectation =
+      [[XCTestExpectation alloc] initWithDescription:@"Result finished"];
+
+  // iPhone 13 Cameras:
```

Comment:
these cameras are not specific to iPhone 13

Reply:
This was copied from the existing tests

Reply:
updated the comments